### PR TITLE
Update contributing guide with links to MSFT style guide

### PR DESCRIPTION
**Pull request by brianamarie**
## Overview
**TL;DR**

This PR adds links to the [MSFT writing style guide](https://docs.microsoft.com/en-us/style-guide/welcome/) and their [top 10 tips](https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice). 

This is prompted by a community contribution that doesn't match previous style, and also the community member pointing out we have no documented reference to what style we strive to adhere to. https://github.com/github/training-kit/pull/824

### Questions
@github/ps-content-and-enablement, does this addition make sense? I know GitHub isn't MSFT, but in absence of a public writing style guide from GitHub, I believe some guidance is far better than no guidance.

### Review
- [ ] Feedback/approval from @github/ps-content-and-enablement  👀 

ps ✨ Thank you @JenniferKerns for introducing me to this manual of style! ✨

### CODE REVIEW

**Comment by brianamarie** [comment]
Thank you so much, @lucie-docs! I've gone ahead and updated this branch with the newest commits on the main branch to see if we can get the build to pass.

This is good to merge, thank you @lucie-docs!

**Review by lucie-docs — APPROVED** [review]
